src/constants/options/status.ts

```diff
@@ -89,3 +89,15 @@ export const getChangeRequestStatusOptions = (t: typeof useT | Function = (t: st
       title: t("No update")
     }
   ] as Option[];
+
+export const getNothingReportOptions = (t: typeof useT | Function = (t: string) => t) =>
+  [
+    {
+      value: "1",
+      title: t("Reported Info")
+    },
+    {
+      value: "-1",
+      title: t("Nothing to Report")
+    }
```

Comment:
Shouldn't these be `true` / `false` (or if strings are required, `"true"`, `"false"`)?

Reply:
I'll pull down your branch and see if I can figure it out either this afternoon or tomorrow morning. Please in the future, if you're needing to spend hours trying to track something like this down, pull me in. That definitely is a case of spending too long trying to figure something out without asking for help.